### ISSUE TITLE
VMware vSphere 7.0 vCenter Appliance Photon OS STIG PHTN-30-000039 Improper file location

### ISSUE THREAD

**Issue by daberkow** [vmware/dod-compliance-and-automation#98]
### Describe the bug

The STIG says to "Open: /etc/rsyslog.conf" and put the example lines in for syslog. I think teh file you want is `/etc/vmware-syslog/syslog.conf` .

### Reproduction steps

```bash
1. Edit /etc/rsyslog.conf like the stig says
2. `systemctl restart rsyslog.service`
3. Rsyslog fails to start
```

### Expected behavior

If you edit `/etc/vmware-syslog/syslog.conf` instead, it is already in the proper format (or is edited by the appliance UI) and then syslog will work.

**Comment by rlakey** [comment]
@daberkow This is something I should remove from the vCenter 7.0 content for Photon since we can configure syslog through the vCenter VAMI.

**Comment by daberkow** [comment]
@rlakey Sounds good! And I appreciate your work on this, just wanted to flag!

**Comment by rlakey** [comment]
No worries keep them coming!  

Fixed by #104
